Library/ViewModels/PledgeAmountViewModel.swift

```diff
@@ -4,6 +4,8 @@ import Prelude
 import ReactiveExtensions
 import ReactiveSwift
 
+public typealias PledgeAmountData = (amount: Double, min: Double, max: Double, isValid: Bool)
```

Comment:
This typealias was previously in `PledgeViewModel` for some reason. I thought it made more sense for it to live here.